src/components/views/messages/MLocationBody.tsx

```diff
@@ -93,16 +95,19 @@ export default class MLocationBody extends React.Component<IBodyProps, IState> {
             bodyId={this.getBodyId()}
             markerId={this.getMarkerId()}
             error={this.state.error}
+            tooltip={_t("Expand map")}
             onClick={this.onClick}
         />;
     }
 }
 
+
```

Comment:
redundant two blank lines?

Reply:
Fixed in 04aebeb0e0ed975a45f49191f8830fca949bda45